models/Document/Tag/Areablock.php

```diff
@@ -54,6 +54,11 @@ class Areablock extends Model\Document\Tag implements BlockInterface
      */
     protected $blockStarted;
 
+    /**
+     * @var array
+     */
+    private $elementTypeUsageCounter = [];
```

Comment:
Maybe here `$brickTypeUsageCounter` to be consistent with the JS code?

Reply:
done!